MovieChecker/MovieChecker/MovieDetailViewController.h

```diff
@@ -0,0 +1,21 @@
+//
+//  MovieDetailViewController.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "Movie.h"
+
+@interface MovieDetailViewController : UITableViewController
+
+@property (nonatomic, weak) Movie *movie;
+
+@property (nonatomic, weak) IBOutlet UILabel *movieTitle;
+@property (nonatomic, weak) IBOutlet UILabel *rating;
+@property (weak, nonatomic) IBOutlet UILabel *synopsis;
+@property (weak, nonatomic) IBOutlet UILabel *year;
```

Comment:
The order of `atomic`/`nonatomic` and `weak`/`strong`/`assign` isn't specified in the standards, but you should be consistent in the way you order them, at least on a per-file basis of not a per-project basis.